addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.xml

```diff
@@ -17,13 +17,13 @@
                             <i t-elif="!user.im_status" class="fa fa-fw fa-question-circle" title="No IM status available"/>
                         </span>
                     </span>
-                    <div class="d-flex flex-column o_card_user_infos">
-                        <span class="fw-bold" t-esc="user.name"/>
+                    <div class="d-flex flex-column o_card_user_infos overflow-hidden">
+                        <span class="fw-bold text-truncate" t-esc="user.name" t-att-title="user ? user.name : record.name"/>
```

Comment:
Hello, I don't know what is happening but it makes no sense to have to check the existence of the user for the title specifically but not for the t-esc? I think this requires more investigation to understand what is going on.

Reply:
Hello @seb-odoo, actually this PR just aimed to apply a temporary fix to prevent an overflow issue [introduced in another PR](https://github.com/odoo/odoo/pull/137915) but I understand your point and I do agree that the basic behaviour was already weird when I jumped on it.

@juliusc2066 I saw that you implemented this, perhaps you could provide us some insights about these lines and why we need to check for `user` in order to escape `user.name` as title attribute ?

Thanks

Reply:
Hello @juliusc2066, small reminder, it'd be nice to have your input here so we can deploy this visual fix, thanks 🙏

Reply:
Hello @chgo-odoo , this is definetly an issue with one of the (many) extensions of the AvatarCardPopover component. I guess that in this case it is with the AvatarCardResourcePopover from resource_mail. You definitely should remove the check inside AvatarCardPopover and replace the title with an x-path inside the AvatarCardResourcePopover template like it is already done with the t-esc
```suggestion
                        <span class="fw-bold text-truncate" t-esc="user.name" t-att-title="user.name"/>
```
and in avatar_card_resource_popover.xml:
```
<xpath expr="//div[hasclass('o_card_user_infos')]/span" position="attributes">
    <attribute name="t-esc">record.name</attribute>
    <attribute name="t-att-title">record.name</attribute>
</xpath>
```
Let me know if there are still issues afterwards :)

Reply:
Seems to fix the issue indeed.. So the issue here was that the AvatarCardResourcePopover from resource_mail didn't know what `user.name` is, right?